### ISSUE TITLE
No more updates for DivestOS and all mobile apps

### ISSUE THREAD

**Issue by robovoice1** [Tobi823/ffupdater#662]
https://divestos.org/pages/news#end

https://infosec.exchange/@divested/113703337476638702

Mull, Mulch

**Comment by Tobi823** [comment]
Thanks for the info, I just read it. 

I will remove Mull and Mulch next January.

**Comment by robovoice1** [comment]
I do not know if Fennec Browser is also affected by this decision.  Tavi is also involved, but no explicit mention of Fennec.
https://gitlab.com/relan/fennecbuild/-/commit/9980c3ecf2ed5c8dfdf6039fba1dbf3890b1f85a

**Comment by TheGerman0** [comment]
@veganomy No, it's build by the F-Droid Team, like all Apps in the official F-Droid Repo. Developers can ask the F-Droid Team to add their Apps in the official Repo and build them, but Mozilla didn't, Fennec is maintained by F-Droid and the Community.

**Comment by IlDivinatore01** [comment]
Why completely remove instead of eol browser section?
